### PR TITLE
[JENKINS-38134] Generate shasum for war, windows and osx installers

### PR DESCRIPTION
**Pull request by armfergom**
[JENKINS-38134](https://issues.jenkins-ci.org/browse/JENKINS-38134)

@reviewbybees

### CODE REVIEW

**Comment by svanoort** [comment]
Have you tested that this will work correctly with the test publish repo?

**Comment by armfergom** [comment]
@svanoort yes

**Comment by svanoort** [comment]
🐝 then

**Comment by recena** [comment]
🐝

**Comment by armfergom** [comment]
@reviewbybees done

File: Makefile [hunk 1]
```diff
@@ -40,7 +40,9 @@ docker.test: docker.images
 war: ${WAR}
 war.publish: ${WAR}
 	ssh ${SSH_OPTS} ${PKGSERVER} mkdir -p "'${WARDIR}/${VERSION}/'"
-	rsync -avz -e "ssh ${SSH_OPTS}" "${WAR}" "${PKGSERVER}:${WARDIR}/${VERSION}/${ARTIFACTNAME}.war"
+	sha256sum ${WAR} > ${WAR_SHASUM}
+	cat ${WAR_SHASUM}
+	rsync -avz -e "ssh ${SSH_OPTS}" "${WAR}" "${WAR_SHASUM}" "${PKGSERVER}:${WARDIR}/${VERSION}/${ARTIFACTNAME}.war"
```
Comment (daniel-beck) [inline Makefile]: This can't possibly have worked, right? Two origin files, and a destination file name?

**Comment by svanoort** [comment]
Concur with @daniel-beck  on this one... would have seemed more reasonable.

**Comment by recena** [comment]
@daniel-beck @svanoort We are following the same convention than Debian community.

**Pull request merged**